cmd/api-server/conureerrors/errors.go

```diff
@@ -60,3 +63,16 @@ func AbortWithError(c *gin.Context, err error) {
 		})
 	}
 }
+
+func AbortWithValidationError(c *gin.Context, err validator.ValidationErrors) {
```

Comment:
I think it's better to move this to the AbortWithError function and check if the error is a `validator.ValidationErrors`. With this, we have only one function to handle the errors. What are your thoughts?